### PR TITLE
BITCON-40 update conviva SDK to the latest version

### PR DESCRIPTION
**Pull request by mostafashaaban5**

### CODE REVIEW

File: ConvivaIntegration/components/conviva/ConvivaClient.brs [hunk 1]
```diff
@@ -844,7 +840,7 @@ function ConvivaClientInstance(settings as object)
 
     adMetadata = {}
     adMetadata.SetModeCaseSensitive()
-    if breakInfo.GetStart() = 0 and self.convivaYoSpaceSession.GetSession()._CLASSNAME <> "YSLiveSession"
+    if breakInfo.GetStart() = 0 and self.convivaYoSpaceSession.GetSession()<> invalid and self.convivaYoSpaceSession.GetSession()._CLASSNAME <> "YSLiveSession"
```
Comment (turbidwater) [inline ConvivaIntegration/components/conviva/ConvivaClient.brs:843]: nitpick: @mostafashaaban5 please put a space in front of the <>

**Review by turbidwater — COMMENTED** [review]
Looks good other than one little whitespace issue

File: ConvivaIntegration/components/conviva/ConvivaClient.brs (hunk 1 above)
Comment (mostafashaaban5) [reply ConvivaIntegration/components/conviva/ConvivaClient.brs:843]: @turbidwater ConvivaClient.brs is not our class as well. I copy pasted their entire classes without doing any formatting

Comment (turbidwater) [reply ConvivaIntegration/components/conviva/ConvivaClient.brs:843]: haha, ok

**Pull request merged**